tfx/examples/cifar10/cifar10_pipeline_native_keras.py

```diff
@@ -0,0 +1,205 @@
+# Lint as: python2, python3
+# Copyright 2019 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""CIFAR10 image classification with transfer learning example using TFX."""
+
+from __future__ import absolute_import
```

Comment:
```
from __future__ import absolute_import
from __future__ import division
from __future__ import print_function
```

Use the same lines

Reply:
It seems all the other examples in example folder write them in three lines. Maybe three line will make it more consistent with other examples? what do you think?